home/screens/ProjectScreen/LegacyLaunchSection.tsx

```diff
@@ -46,40 +45,36 @@ export function LegacyLaunchSection({ app }: { app: ProjectPageApp }) {
   } else if (isLatestLegacyPublishDeprecated) {
     warning = (
       <WarningBox
-        title="Unsupported SDK version"
-        message={`This project's SDK version (${legacyUpdatesSDKMajorVersion}) is no longer supported.`}
+        message={`The latest classic update on the 'default' release channel uses SDK (${legacyUpdatesSDKMajorVersion}), which is no longer supported.`}
       />
     );
   }
 
   const theme = useExpoTheme();
 
   return (
-    <>
+    <View>
+      <SectionHeader header="Classic Release Channels" style={{ paddingTop: 0 }} />
```

Comment:
```suggestion
      <SectionHeader header="Classic release channels" style={{ paddingTop: 0 }} />
```
Only buttons are capitalized